WooCommerce/Classes/ViewRelated/Editor/AztecEditorViewController.swift

```diff
@@ -242,9 +242,7 @@ extension AztecEditorViewController {
     }
 
     private func presentBackNavigationActionSheet() {
-        UIAlertController.presentSaveChangesActionSheet(viewController: self, onSave: { [weak self] in
-            self?.saveButtonTapped()
-        }, onDiscard: { [weak self] in
+        UIAlertController.presentDiscardChangesActionSheet(viewController: self, onDiscard: { [weak self] in
```

Comment:
Re: `weak self`

Dumb question: Is it ever possible for `self` to be dismissed even though there's a modal `UIAlertController`? 😅

Reply:
Yes, it's always possible for `self` to be dismissed, also if a UIAlertController is presented 👍